.gitignore

```diff
@@ -3,3 +3,4 @@
 *.jl.mem
 deps/deps.jl
 Manifest.toml
+.vscode
```

Comment:
BTW, you can set up global gitignore.

Reply:
Thanks for the tip, should I revert this file?

Reply:
Nah, it's fine.